package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@datorama/akita",
   "version": "4.7.0",
   "description": "State Management Tailored-Made for JS Applications",
-  "schematics": "./schematics/collection.json",
+  "schematics": "./schematics/src/collection.json",
```

Comment:
Why did you change it?

Reply:
The path was not correct. The collection.json is inside `schematics/src/`. When linking @datorama/akita I couldn't use the schematics.

Reply:
When we build the project, this is the correct path.

Reply:
Oh, I see. Sorry about that. I was only building the schematics subfolder when developing I didn't realize. I change that.